src/Mvc/Mvc.Testing/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 #nullable enable
+Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactory<TEntryPoint>.DisposeAsync() -> System.Threading.Tasks.ValueTask
+virtual Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactory<TEntryPoint>.DisposeAsyncCore() -> System.Threading.Tasks.ValueTask
```

Comment:
@javiercn can we make sure this API isn't part of the final result. Also the implementation might need some ❤️